WcaOnRails/db/migrate/20160701034833_reference_table_updates.rb

```diff
@@ -0,0 +1,67 @@
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333bf', '3', 1), ('333bf', '2', 2), ('333bf', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333oh', 'a', 1), ('333oh', '3', 2), ('333oh', '2', 3), ('333oh', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333fm', 'm', 1), ('333fm', '2', 2), ('333fm', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333ft', 'm', 1), ('333ft', '2', 2), ('333ft', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('minx', 'a', 1), ('minx', '3', 2), ('minx', '2', 3), ('minx', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('pyram', 'a', 1), ('pyram', '3', 2), ('pyram', '2', 3), ('pyram', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('sq1', 'a', 1), ('sq1', '3', 2), ('sq1', '2', 3), ('sq1', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('clock', 'a', 1), ('clock', '3', 2), ('clock', '2', 3), ('clock', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('skewb', 'a', 1), ('skewb', '3', 2), ('skewb', '2', 3), ('skewb', '1', 4);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('666', 'm', 1), ('666', '2', 2), ('666', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('777', 'm', 1), ('777', '2', 2), ('777', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('444bf', '3', 1), ('444bf', '2', 2), ('444bf', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('555bf', '3', 1), ('555bf', '2', 2), ('555bf', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333mbf', '3', 1), ('333mbf', '2', 2), ('333mbf', '1', 3);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('magic', 'a', 1);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('mmagic', 'a', 1);")
+    ActiveRecord::Base.connection.execute("insert into preferred_formats (event_id, format_id, ranking) values ('333mbo', 'a', 1);")
+  end
```

Comment:
This can be cleaner
```ruby
{ "333" => %(a 3 2 1),
  "444" => %(a 3 2 1),
  ... # the rest
}.each do |event_id, format_ids|
  format_ids.each_with_index do |format_id, i|
    Format.create(event_id: event_id, format_id: format_id, ranking: (i + 1))
  end
end
```